assets/js/components/Dashboard/AlertsPage.tsx

```diff
@@ -14,7 +126,67 @@ const AlertsPage: ComponentType<Props> = (props: Props) => {
     >
       <div className="page-content__header">Posted Alerts</div>
       <div className="page-content__body">
-        <div style={{ color: "white" }}>This will be posted alerts</div>
+        <Container fluid>
+          <Row className="place-list__header-row">
+            <Col lg={3}>
+              <div
+                className="place-list__sort-label d-flex align-items-center"
+                onClick={sortLabelOnClick}
+                data-testid="sort-label"
+              >
+                {alertSortLabel}
+                {alertSortDirection === 0 ? <ArrowDown /> : <ArrowUp />}
+              </div>
+            </Col>
+            <Col lg={3} className="d-flex justify-content-end pe-3">
+              <FilterDropdown
+                list={MODES_AND_LINES}
+                onSelect={(value: any) => handleAlertModeOrLineSelect(value)}
+                selectedValue={alertModeLineFilterValue}
+                className="modes-and-lines"
+              />
+            </Col>
+            <Col lg={3} className="place-screen-types pe-3">
+              <FilterDropdown
+                list={SCREEN_TYPES}
+                onSelect={(value: any) => handleAlertScreenTypeSelect(value)}
+                selectedValue={alertScreenTypeFilterValue}
+                className="screen-types"
+              />
+            </Col>
+            <Col lg={3}>
+              <FilterDropdown
+                list={STATUSES}
+                onSelect={(value: any) => handleAlertStatusSelect(value)}
+                selectedValue={alertStatusFilterValue}
+                className="statuses"
+                disabled
+              />
+            </Col>
+          </Row>
+        </Container>
+        {filterAlerts()
+          .sort((a: Alert, b: Alert) => {
+            return alertSortDirection === 0
+              ? compareAlerts(a, b)
+              : compareAlerts(b, a);
+          })
```

Comment:
Ah, I realize the original commit I made was not correct because it didn't include def of `a` and `b`. Can still consolidate this, though
```suggestion
          .sort((a: Alert, b: Alert) =>
            alertSortDirection === 0 ? compareAlerts(a, b) : compareAlerts(b, a)
          )
```